crates/evm/core/src/fork/backend.rs

```diff
@@ -141,8 +142,10 @@ where
                 trace!(target: "backendhandler", "received request basic address={:?}", addr);
```

Comment:
better to have addr as named field

Reply:
Yes, but that's not a part of the diff or relevant to the pr. 
Lots of improvements to tracing and error handling that could be made throughout the fork.